plugin/pxr/maya/plugin/pxrUsd/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ pxr_plugin(${PXR_PACKAGE}
 
     CPPFILES
         alembicChaser.cpp
+        infoImportChaser.cpp
```

Comment:
You added the test to the common location but made it dependent on building the Pixar plugin. This dependency we want to avoid and to achieve it, I propose to move infoImportChaser to https://github.com/Autodesk/maya-usd/tree/dev/test/lib/usd/plugin

Reply:
I'd like to avoid it as well, but the problem is that if I move it, I run into the same problem that I described in the PR description:

> I ran into an issue that I haven't yet been able to find time to dig into yet, which is that it appears that the pxrUsd plugin must be loaded first before the mayaUsdPlugin, or else neither export nor import chasers will be able to register when being called from the respective mayaUSDImport and mayaUSDExport commands (the error Failed to create import chaser is shown because somehow the macro that registers the chaser is never executed; this also happens even for the default Alembic export chaser plugin). If someone from ADSK or elsewhere could help investigate, I'd appreciate it.

Internally, we do not register/build our chasers the same way (they are separate DLLs altogether), so we do not have this issue. But for the sake of having a working example that anyone can compile and use, I left it in the same location as the Alembic chaser for now.

Reply:
What I was suggesting is making your test chaser a new plugin, so this very much resembles your setup with separate DLLs. Have a look at what's in https://github.com/Autodesk/maya-usd/tree/dev/test/lib/usd/plugin.

Reply:
This request has been accommodated in 271c742040c5e2c3feeebd96831fefffc3e8eb2a.

Reply:
I see you added the chaser to one of the existing plugins. I must have misguided you, but my hope was you will add a new target library to https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/plugin/CMakeLists.txt that will only have your test chaser plugin.

As for test setup, see how `MAYA_PLUG_IN_PATH` is passed when registering a test https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/translators/CMakeLists.txt#L216

Reply:
> I see you added the chaser to one of the existing plugins. I must have misguided you, but my hope was you will add a new target library to https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/plugin/CMakeLists.txt that will only have your test chaser plugin.
Ah, I see. I have accommodated for this request in 57d630d1e. Hopefully that's correct now, though I must admit this setup is a little confusing to me, so I might still be misunderstanding. Please let me know if things look appropriate now.

> As for test setup, see how MAYA_PLUG_IN_PATH is passed when registering a test https://github.com/Autodesk/maya-usd/blob/dev/test/lib/usd/translators/CMakeLists.txt#L216
I did glance at that file, but I don't see immediately how it adds the plugins in `maya-usd/build/test/lib/usd/plugin/Debug` to the `MAYA_PLUG_IN_PATH`; the fact that I cannot run the existing test `testUsdMayaListShadingModesCommand` either with the same plugin import issue implies to me that it's not just the new test I'm adding that has the issue.

Reply:
Have a look at the attached diff - this is how you configure test env when a particular test has special dependencies.
[fix_test_import_chaser.diff.txt](https://github.com/Autodesk/maya-usd/files/6021570/fix_test_import_chaser.diff.txt)

Reply:
Hmm, it still doesn't work for me. I think that's because when I build and test, I'm always doing so in `Debug` configuration, which means my plugins get deployed here:

`maya-usd/build/test/lib/usd/translators/../plugin/Debug/`

I've gone ahead and applied your patch in `ca1d981bc`, regardless.